lessons/introduction-to-smartpy/08/08.mdx

```diff
@@ -84,7 +84,7 @@ Here, SmartPy automatically determines and remembers that `name` is a variable t
 
 Just to practise what you learnt above: 
 
-**In `Plant` contract shown to your left, what is the type of `name` that SmartPy will automatically infer?**  
+**In `Plant` contract shown to your left, what is the data type of `name` that SmartPy will automatically infer?**  
```

Comment:
are you referring to the contract inside the editor? if that's the case, shouldn't it be "to your right"

Reply:
Good catch! Fixed.